src/main/java/com/waytta/clientinterface/LocalClient.java

```diff
@@ -1,13 +1,60 @@
 package com.waytta.clientinterface;
 
+import hudson.Extension;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+import hudson.util.FormValidation;
+import org.jenkinsci.Symbol;
+
+import com.waytta.Utils;
+
 public class LocalClient extends BasicClient {
+    public static final int DEFAULT_JOB_POLL_TIME = 10;
+    private Integer jobPollTime = DEFAULT_JOB_POLL_TIME;
+    private Boolean blockbuild = false;
+    private String target;
+    private String targetType;
 
-    public LocalClient(String credentialsId, String target, String targettype, String function, Boolean blockbuild, Integer jobPollTime) {
-        super(credentialsId, target, targettype, function);
+    @DataBoundConstructor
+    public LocalClient(Boolean blockbuild, Integer jobPollTime, String target, String targetType) {
```

Comment:
Use primitives. For the actual fields, use primitives unless you really need to handle null values, typically only necessary for compatibly upgrading from an earlier version lacking the field, in which case `readResolve` would set a non-null value.